lib/api/apiUtils/authorization/permissionChecks.js

```diff
@@ -385,10 +387,19 @@ function processBucketPolicy(requestType, bucket, canonicalID, arn, bucketOwner,
  * @returns {boolean} - true if the quota is valid, false otherwise
  */
 async function validateQuotas(bucket, account, apiNames, apiMethod, inflight, log, callback) {
+    const requestStartTime = process.hrtime.bigint();
     const bucketQuota = bucket.getQuota();
     const accountQuota = account?.quota || 0;
     let bucketQuotaExceeded = false;
     let accountQuotaExceeded = false;
+    let type = 'bucket';
+    if (bucketQuota && accountQuota) {
+        type = 'bucketAccount';
```

Comment:
probably should make it parsable: `bucket+account` or `bucket,account` ?
(or use separate labels, c.f. other comment)

Reply:
I will make it parsable indeed, will choose the first option

Reply:
See https://github.com/scality/cloudserver/pull/5556/files#diff-773344fcfc0f64145ff95eb6eccd556588c19449dc2c263a2532d4030bf00dffR407